Makefile

```diff
@@ -1,5 +1,6 @@
 PREFIX    ?= /usr/local
 MANPREFIX ?= $(PREFIX)/share/man
+LOCALE_PREFIX ?= /usr/share/locale
```

Comment:
If we decide not to fix the `TEXTDOMAINDIR` packaging bug, I think maybe we don't use an override-able variable here (overriding it would just break `downgrade`). I would just hard-code the same path again wherever you're using it.

Reply:
Will fix this with my suggestion for `downgrade.in`